plugin/pkg/scheduler/api/types.go

```diff
@@ -43,6 +43,7 @@ type PriorityPolicy struct {
 	// For the Kubernetes provided priority functions, the name is the identifier of the pre-defined priority function
 	Name string `json:"name"`
 	// The numeric multiplier for the minion scores that the priority function generates
+	// The weight should be non-zero and can be a positive or a negative integer
```

Comment:
One nit there: https://github.com/openshift/openshift-docs/blob/master/using_openshift/scheduler.adoc#step-2-prioritize-the-filtered-list-of-nodes

says the range is 0-10, but really the range should be -10 through 10 excluding 0. (since you allow negative weights)